app/src/main/java/org/stepik/android/domain/course/repository/CourseReviewSummaryRepository.kt

```diff
@@ -1,8 +1,10 @@
 package org.stepik.android.domain.course.repository
 
 import io.reactivex.Single
+import org.stepik.android.domain.base.DataSourceType
 import org.stepik.android.model.CourseReviewSummary
 
 interface CourseReviewSummaryRepository {
     fun getCourseReviewSummary(courseReviewSummaryId: Long): Single<CourseReviewSummary>
```

Comment:
Implementation of this method can be replaced with usage of `getCourseReviewSummaries`